<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QMainWindow" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2112</width>
    <height>1452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="rect_x">
    <property name="geometry">
     <rect>
      <x>1530</x>
      <y>90</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rect_y">
    <property name="geometry">
     <rect>
      <x>1630</x>
      <y>90</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rect_w">
    <property name="geometry">
     <rect>
      <x>1730</x>
      <y>90</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rect_h">
    <property name="geometry">
     <rect>
      <x>1860</x>
      <y>90</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rect_mass">
    <property name="geometry">
     <rect>
      <x>1990</x>
      <y>90</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1560</x>
      <y>60</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1660</x>
      <y>60</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1750</x>
      <y>60</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Width</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1880</x>
      <y>60</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Height</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>2010</x>
      <y>60</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Mass</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1580</x>
      <y>250</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1730</x>
      <y>250</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>1850</x>
      <y>250</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Radius</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>2000</x>
      <y>250</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Mass</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="circle_x">
    <property name="geometry">
     <rect>
      <x>1540</x>
      <y>290</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>500</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="circle_y">
    <property name="geometry">
     <rect>
      <x>1680</x>
      <y>290</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>500</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="circle_r">
    <property name="geometry">
     <rect>
      <x>1830</x>
      <y>290</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>50</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="circle_mass">
    <property name="geometry">
     <rect>
      <x>1980</x>
      <y>290</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>50</string>
    </property>
   </widget>
   <widget class="QPushButton" name="circle_butt">
    <property name="geometry">
     <rect>
      <x>1660</x>
      <y>350</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить окружность</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1650</x>
      <y>140</y>
      <width>331</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить прямоугольник</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="paintOn">
    <property name="geometry">
     <rect>
      <x>1560</x>
      <y>450</y>
      <width>411</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Отображать фигуры</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>1540</x>
      <y>700</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Характеристики системы:</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>500</y>
      <width>611</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>1550</x>
      <y>770</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Общий периметр:</string>
    </property>
   </widget>
   <widget class="QLabel" name="perimeter">
    <property name="geometry">
     <rect>
      <x>1750</x>
      <y>770</y>
      <width>301</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>1550</x>
      <y>820</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Общая площадь:</string>
    </property>
   </widget>
   <widget class="QLabel" name="square">
    <property name="geometry">
     <rect>
      <x>1730</x>
      <y>820</y>
      <width>331</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>1490</x>
      <y>0</y>
      <width>20</width>
      <height>1481</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>1550</x>
      <y>870</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Занимаемая память:</string>
    </property>
   </widget>
   <widget class="QLabel" name="memory">
    <property name="geometry">
     <rect>
      <x>1770</x>
      <y>870</y>
      <width>331</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>1550</x>
      <y>920</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Центр масс:</string>
    </property>
   </widget>
   <widget class="QLabel" name="center">
    <property name="geometry">
     <rect>
      <x>1690</x>
      <y>920</y>
      <width>411</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0 ; 0</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>660</y>
      <width>611</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>1540</x>
      <y>540</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Syberian Mouse:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>1560</x>
      <y>600</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Координаты:</string>
    </property>
   </widget>
   <widget class="QLabel" name="mouse_coord">
    <property name="geometry">
     <rect>
      <x>1700</x>
      <y>600</y>
      <width>341</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0 ; 0</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="rect_check">
    <property name="geometry">
     <rect>
      <x>1570</x>
      <y>150</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="circle_check">
    <property name="geometry">
     <rect>
      <x>1570</x>
      <y>360</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2112</width>
     <height>38</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>circle_butt</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>addCircle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1863</x>
     <y>428</y>
    </hint>
    <hint type="destinationlabel">
     <x>2073</x>
     <y>382</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>addRect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1842</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>2065</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>paintOn</sender>
   <signal>stateChanged(int)</signal>
   <receiver>Widget</receiver>
   <slot>changePaint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1776</x>
     <y>525</y>
    </hint>
    <hint type="destinationlabel">
     <x>2077</x>
     <y>521</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addRect()</slot>
  <slot>addCircle()</slot>
  <slot>changePaint()</slot>
 </slots>
</ui>
